pkg/api/legacy/authorization.go

```diff
@@ -0,0 +1,98 @@
+package legacy
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+
+	authorizationv1 "github.com/openshift/api/authorization/v1"
+	"github.com/openshift/origin/pkg/authorization/apis/authorization"
+	authorizationv1helpers "github.com/openshift/origin/pkg/authorization/apis/authorization/v1"
+)
+
+// InstallLegacyAuthorization this looks like a lot of duplication, but the code in the individual versions is living and may
+// change. The code here should never change and needs to allow the other code to move independently.
+func InstallLegacyAuthorization(scheme *runtime.Scheme) {
```

Comment:
add a comment describing when someone should use this?

Reply:
> add a comment describing when someone should use this?

Never, they should never use this.  If you think you should use this, reconsider for the sake of your happiness.  People should only remove uses of this.

Reply:
> add a comment describing when someone should use this?

doc'd the package as a whole.